### PR TITLE
[DynamicDelivery] Advance section

### PR DESCRIPTION
**Pull request by Mercandj**

### CODE REVIEW

File: mwm-5-dynamic-delivery/presentation/PITCHME.md [hunk 1]
```diff
@@ -755,15 +757,55 @@ Resource shrinker cannot be used for multi-apk applications
...
 
+Note:
+
+- You can choose which dynamic features to include when deploying your app
+
 ---?image=mwm-5-dynamic-delivery/presentation/run-debug-2.png&size=auto 80%
 
+Note:
+
+- By default, Android Studio doesn’t use app bundles to deploy your app—it builds and pushes APKs to your device that are optimized for deployment speed, rather than APK size. To configure Android Studio to instead build and deploy APKs from an app bundle, modify your run/debug configuration and select the option to deploy APK from app bundle.
```
Comment (bowserf) [inline mwm-5-dynamic-delivery/presentation/PITCHME.md]: app`—`it ?

File: mwm-5-dynamic-delivery/presentation/PITCHME.md [hunk 2]
```diff
@@ -755,15 +757,55 @@ Resource shrinker cannot be used for multi-apk applications
...
+### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Code </span><span style="text-transform: none; font-size:0.8em;"> shrinking</span>
+
+- To enable code shrinking for entire project, you must do so from the base module’s build.gradle file
+- MinifyEnabled property in dynamic feature module build configurations is ignored
+
+---
+
+### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Version</span>
+
+- You base module and dynamic feature apks will share the same version code and version name
```
Comment (bowserf) [inline mwm-5-dynamic-delivery/presentation/PITCHME.md]: You`r`

File: mwm-5-dynamic-delivery/presentation/PITCHME.md [hunk 3]
```diff
@@ -755,15 +757,55 @@ Resource shrinker cannot be used for multi-apk applications
...
+- The base module version code and version name is the reference
+
+---
+
+### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Open </span><span style="text-transform: none; font-size:0.8em;"> question</span>
+
+- Cascade of dynamic module?
```
Comment (bowserf) [inline mwm-5-dynamic-delivery/presentation/PITCHME.md]: Plural !

File: mwm-5-dynamic-delivery/presentation/PITCHME.md [hunk 4]
```diff
@@ -755,15 +757,55 @@ Resource shrinker cannot be used for multi-apk applications
...
+---
+
+### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Go</span><span style="text-transform: none; font-size:0.8em;"> further</span>
+
+---
+
+### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Clear</span><span style="text-transform: none; font-size:0.8em;"> app storage from settings</span>
+
+- Makes `splitInstallManager.installedModules.contains(searchgModuleName)` returning false
+- Does not uninstall apk, the next `startInstall(request)` is almost synchronous
```
Comment (bowserf) [inline mwm-5-dynamic-delivery/presentation/PITCHME.md]: Ballzy =p

Comment (Mercandj) [reply mwm-5-dynamic-delivery/presentation/PITCHME.md]: Ohhh yeah

**Pull request merged**